webrtc.html

```diff
@@ -4076,35 +4076,73 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <code><a>RTCSessionDescription</a></code> may change it to a
               non-null value, as defined in <span data-jsep=
               "processing-a-local-desc processing-a-remote-desc">[[!JSEP]]</span>.</p>
-              <p>If a track is passed in, the value of the
-              <code>sender.track</code> will be set to that track and the MSID
-              and media type generated by <code>createOffer</code> will be that
-              of the track.</p>
-              <p>If a kind is passed in and the value is not a legal
-              <code><a>MediaStreamTrack</a></code> <code>kind</code>, throw a
-              <code>TypeError</code>.</p>
-              <p>If a kind is passed in, the value of the
-              <code>sender.track</code> will be null and and media type
-              generated by <code>createOffer</code> will be that of the kind.
-              The MSID generated by <code>createOffer</code> (if necessary,
-              such as when <code>init.send == true</code>) will be selected by
-              the user agent and will not be related to any track. Future calls
-              to <code>sender.replaceTrack</code> with a track of a different
-              kind will fail. Future calls will not change the MSID associated
-              with the transceiver.</p>
-              <p>If <code>init.sendEncodings</code> is set, then subsequent
-              calls to <code>createOffer</code> will be configured to send with
-              multiple RTP encodings as defined in <span data-jsep=
-              "subsequent-offers">[[!JSEP]]</span>. When
-              <code>setRemoteDescription</code> is called with a corresponding
-              remote description that is able to receive multiple RTP encodings
-              as defined in [[!JSEP]], the <code><a>RTCRtpSender</a></code> may
-              send multiple RTP encodings and the parameters retrieved via the
-              transceiver&#8217;s <code>sender.getParameters()</code> will
-              reflect the encodings negotiated.</p>
-              <p>RID values passed into <code>init.sendEncodings</code> must be
-              composed only of case-sensitive alphanumeric characters (a-z,
-              A-Z, 0-9) up to a maximum of 16 characters.</p>
+              <p>When this method is invoked, the User Agent must run the
+              following steps:</p>
+              <ol>
+                <li>
+                  <p>If the dictionary argument is present, and it has a
+                  streams member, let <var>streams</var> be that list of <code>
```

Comment:
I'd put the name of the members between `<code></code>` (here and for `sendEncodings` as well)